src/sentry/api/event_search.py

```diff
@@ -106,9 +106,10 @@
 # "              # quote literal
 quoted_value = ~r"\"((?:[^\"]|(?<=\\)[\"])*)?\""s
 
-# Format tokens
-date_format          = ~r"\d{4}-\d{2}-\d{2}(T\d{2}:\d{2}:\d{2}(\.\d{1,6})?)?Z?(?=\s|\)|$)"
-alt_date_format      = ~r"\d{4}-\d{2}-\d{2}(T\d{2}:\d{2}:\d{2}(\.\d{1,6})?(\+\d{2}:\d{2})?)?(?=\s|\)|$)"
+# Formats
+date_time_1          = ~r"\d{4}-\d{2}-\d{2}(T\d{2}:\d{2}:\d{2}(\.\d{1,6})?)?Z?(?=\s|\)|$)"
+date_time_2          = ~r"\d{4}-\d{2}-\d{2}(T\d{2}:\d{2}:\d{2}(\.\d{1,6})?(\+\d{2}:\d{2})?)?(?=\s|\)|$)"
+date_time_format     = (date_time_1 / date_time_2)
```

Comment:
Is there anywhere we actually use these separately? Maybe they should just be directly in `date_time_format`?

Reply:
I like Evan's suggestion of giving them names, that is a good argument for keeping them separate